onnxscript/rewriter/_ir_utils.py

```diff
@@ -29,12 +27,3 @@ def propagate_const_value(ir_value: ir.Value) -> ir.Value:
                     ir_value.const_value = attr_value.value  # type: ignore[union-attr]
                     break
     return ir_value
-
-
-def get_numpy_from_ir_value(value: ir.Value) -> np.ndarray | None:
-    constant_value = value.const_value
-    if constant_value is not None:
-        if isinstance(constant_value, ir.serde.TensorProtoTensor):
-            return constant_value.numpy()
-        return np.array(constant_value)
```

Comment:
@justinchuby based on your comment, sounds like it's this line we don't want. We should delete the np.array() call and keep this function?